CHANGELOG.md

```diff
@@ -0,0 +1,114 @@
+# Changelog
+
+## [0.4.0](https://github.com/voxpupuli/puppet-mode/tree/0.4.0) (2019-03-10)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-mode/compare/0.3...0.4.0)
+
+**Implemented enhancements:**
+
+- Indentation style closer to puppet-lint style [\#51](https://github.com/voxpupuli/puppet-mode/issues/51)
+- Font-lock string matches in a case as strings [\#48](https://github.com/voxpupuli/puppet-mode/issues/48)
+- Change : syntax to symbol [\#38](https://github.com/voxpupuli/puppet-mode/issues/38)
+- Fontify arguments to require and include [\#37](https://github.com/voxpupuli/puppet-mode/issues/37)
+- Prevent comments from changing the indentation level [\#113](https://github.com/voxpupuli/puppet-mode/pull/113) ([paran1](https://github.com/paran1))
+- Update keywords, functions and metaparameters [\#106](https://github.com/voxpupuli/puppet-mode/pull/106) ([paran1](https://github.com/paran1))
+- Fix indentation of closing braces [\#104](https://github.com/voxpupuli/puppet-mode/pull/104) ([paran1](https://github.com/paran1))
+- Uniform region alignment [\#55](https://github.com/voxpupuli/puppet-mode/pull/55) ([russell](https://github.com/russell))
+- Power up docs [\#32](https://github.com/voxpupuli/puppet-mode/pull/32) ([lunaryorn](https://github.com/lunaryorn))
+
+**Fixed bugs:**
+
+- incorrect indentation when using -\> [\#65](https://github.com/voxpupuli/puppet-mode/issues/65)
+- Set require-final-line to the value of mode-require-final-line, not t [\#52](https://github.com/voxpupuli/puppet-mode/issues/52)
+- Trailing colon in a comment causes an indent [\#49](https://github.com/voxpupuli/puppet-mode/issues/49)
+- Applying manifests over tramp [\#46](https://github.com/voxpupuli/puppet-mode/issues/46)
+- Regexp font-locking issue [\#42](https://github.com/voxpupuli/puppet-mode/issues/42)
+- Array indentation seems broken [\#33](https://github.com/voxpupuli/puppet-mode/issues/33)
+- Add pcase fix for emacs 24.1 [\#50](https://github.com/voxpupuli/puppet-mode/pull/50) ([kmosher](https://github.com/kmosher))
+
+**Closed issues:**
```

Comment:
Admittedly i've fallen off the wagon with the changelog generator thing, but this seems like it should be in the ehancements section. Who decides which changes should be in which section?